OPHD/States/MapViewStateIO.cpp

```diff
@@ -566,6 +567,8 @@ void MapViewState::readPopulation(NAS2D::Xml::XmlElement* element)
 		mCurrentMorale = dictionary.get<int>("morale");
 		mPreviousMorale = dictionary.get<int>("prev_morale");
 
+		mTurnNumberOfLanding = dictionary.get<int>("turn_number_of_landing");
```

Comment:
I've added a default of `constants::ColonyShipOrbitTime`. For posterity, here's my explanation from Discord:

> This will work correctly for people who are importing a save that's older than `constants::ColonyShipOrbitTime + gracePeriod`. It will also work correctly for people who are importing a save where they haven't landed yet.
> 
> Saves who have landed, but haven't hit colony deorbit, will start their grace period immediately even if they have already had some morale/crime changes. Saves who have exceeded colony deorbit will start their grace period immediately but it will end at `constants::ColonyShipOrbitTime + gracePeriod`.

Reply:
This appears to address the concern about a default value quite nicely.